### PR TITLE
OS 204 Cassandra index changes

### PR DESCRIPTION
**Pull request by Hari-stackroute**
Cassandra index changes(unique and composite index)
Miscellaneous fixes

### CODE REVIEW

File: java/registry/src/main/java/io/opensaber/registry/sink/JanusGraphStorage.java [hunk 1]
```diff
@@ -4,31 +4,54 @@
...
+		/*config.setProperty("jdbc.url", connectionInfo.getUri());
 		config.setProperty("jdbc.username", connectionInfo.getUsername());
-		config.setProperty("jdbc.password", connectionInfo.getPassword());
-		config.setProperty("storage.backend", "cassandrathrift");
+		config.setProperty("jdbc.password", connectionInfo.getPassword());*/
+		config.setProperty("storage.backend", "cql");
+		//config.setProperty("query.batch", true);
+
+		//String host = environment.getProperty("cassandra.hostname");
+		config.setProperty("storage.hostname", "127.0.0.1");
```
Comment (indrajra) [inline java/registry/src/main/java/io/opensaber/registry/sink/JanusGraphStorage.java]: IP address is hard-coded. Also, remove commented code. Why remove query.batch?

File: java/registry/src/main/java/io/opensaber/registry/sink/JanusGraphStorage.java [hunk 2]
```diff
@@ -52,10 +75,60 @@ public JanusGraphStorage(Environment environment) {
...
+		VertexLabel vlabel = ((JanusGraph) graph).getVertexLabel(label);
+		graph.tx().commit();
+		JanusGraphManagement janusGraphManagement = ((JanusGraph) graph).openManagement();
+		propertyNames.forEach(propertyName -> {
+			PropertyKey propertyKey = janusGraphManagement.getPropertyKey(propertyName);
+			JanusGraphIndex graphIndex = janusGraphManagement.buildIndex(vlabel.name() + propertyKey.toString(), Vertex.class).addKey(propertyKey).buildCompositeIndex();
+			graphIndexList.add(graphIndex);
+		});
+		janusGraphManagement.commit();
+		//Reindex the existing data
```
Comment (indrajra) [inline java/registry/src/main/java/io/opensaber/registry/sink/JanusGraphStorage.java]: Why this is commented? If it is not needed, pls remove.

File: java/registry/src/main/java/io/opensaber/registry/sink/JanusGraphStorage.java (hunk 1 above)
Comment (Hari-stackroute) [reply java/registry/src/main/java/io/opensaber/registry/sink/JanusGraphStorage.java]: removed the hard-coded value.

File: java/registry/src/main/java/io/opensaber/registry/sink/JanusGraphStorage.java (hunk 2 above)
Comment (Hari-stackroute) [reply java/registry/src/main/java/io/opensaber/registry/sink/JanusGraphStorage.java]: removed

**Pull request merged**